.circleci/config.yml

```diff
@@ -107,26 +120,60 @@ jobs:
       - run: sudo make -C build install
       - run: make -C build package
 
+  build-arm-release:
```

Comment:
and this is the new CI task for building on arm. notice it uses even newer ubuntu (which is what forced the issues with venv). we need newer ubuntu because it has better support for arm packages (and we dont want to have to build our deps from source because that is a huge pain)